README.md

```diff
@@ -22,14 +19,10 @@
 **Basic Installation**
 =====================================
 
-1.  Download everything or clone a copy to your system.
-2.  Install make and cmake.
-3.  Extract the Downloaded file to a directory of your choice (/vegastrike). 
-4.  Enter /vegastrike
-5.  Run command `sh/vsbuild.sh`
-6.  Run command `bin/vegastrike`
-    or double-click the executable file in the bin directory.
-7.  Change settings with command `bin/vssetup`
+Either install Vega Strike from the binary installer for your platform, or follow the instructions for compiling from source. (`Compiling Vegastrike`, below.)
+
+- Run command `bin/vegastrike` or double-click the executable file in the bin directory.
+- Change settings with command `bin/vssetup`
```

Comment:
these should probably be reversed - check setup first, then run vegastrike.

Reply:
Agreed

Reply:
Agreed.

Reply:
@nabaco @BenjamenMeyer The above change has been implemented.